src/cloud/enrichments/README.md

```diff
@@ -377,6 +392,23 @@ This file contains gaze data in world camera coordinates. For a definition of th
 | **gaze x [px]** | Float value representing the x-coordinate of the mapped gaze point in world camera pixel coordinates.
 | **gaze y [px]** | Same as "gaze x [px]" but for the y-coordinate.     |
 | **worn** | This value indicates whether the Pupil Invisible Glasses have been worn by a subject at this point in time. `1.0` indicates that it has been worn, while `0.0` indicates that is has not been. Added in version 2 of this enrichment.    |
+| **fixation index** | If this gaze samples belongs to a fixation event, this is the corresponding index of the fixation. Otherwise this field is empty.     |
+
+
+#### fixations.csv
+This file contains fixation events detected in the gaze data stream.
+
+
+| Field | Description | 
+| -------- | -------- | 
+| **section id** | Unique identifier of the corresponding section.     |
+| **recording id** | Unique identifier of the recording this sample belongs to.     |
+| **fixation index** | Index of fixation within the section.     |
+| **start timestamp [ns]** | UTC timestamp in nanoseconds of the start of the fixation.     |
+| **end timestamp [ns]** | UTC timestamp in nanoseconds of the end of the fixation.     |
+| **duration [ms]** | Duration of the fixation in milliseconds.     |
```

Comment:
There are other interesting values like velocity etc. should we include those?

Reply:
Yes, I agree we should add those!

Reply:
> "mean_vel": 2264.467235822762,
> "peak_vel": 3272.5649504693665
Those values are based on the compensated gaze data (compensated for head movement using optical flow) and are somewhat difficult to understand. One could understand it as the movement speed of the eye while disregarding VOR movements. And it is in pixel space, so hard to relate physiological values. I wonder if we should drop those.


> "label": "gap"
Gap events are basically the time periods between fixations. Mostly those should be saccades, but could also be smooth pursuit movements or false-negative detections due to high noise in the gaze signal. I think there is value in reporting these, as mostly this corresponds well to saccades, but it makes things a bit more complicated. We would probably have to e.g. talk about event IDs rather than fixation IDs then.

Reply:
We decided to not include amplitude and velocity values. Further, we are not including gap events.